File: src/main/java/me/arasple/mc/traction/acts/ActionTell.java
package me.arasple.mc.traction.acts;

import me.arasple.mc.traction.base.AbstractAction;
import me.arasple.mc.trmenu.utils.Vars;
import org.bukkit.entity.Player;

/**
 * @author Arasple
 * @date 2019/12/21 20:34
 */
public class ActionTell extends AbstractAction {

    @Override
    public String getName() {
        return "talk|message|send|tell";
    }

    @Override
    public void onExecute(Player player) {
        player.sendMessage(Vars.replace(player, getContent()));
    }

}

File: src/main/kotlin/me/arasple/mc/trmenu/modules/action/impl/ActionTell.kt
package me.arasple.mc.trmenu.modules.action.impl

import me.arasple.mc.trmenu.modules.action.base.Action
import org.bukkit.entity.Player

/**
 * @author Arasple
 * @date 2020/2/27 9:02
 */
class ActionTell : Action("tell|send|message|msg|talk|say") {

    override fun onExecute(player: Player) = getContentSplited(player).forEach { player.sendMessage(it) }

}